downstream/modules/hub/proc-pushing-container-images-from-your-local.adoc

```diff
@@ -33,27 +32,24 @@ embedded.
 --tls-verify=false
 ----
 +
-. Navigate to the automation hub and click on *Execution Environments* if that
-window is not open.
+. Navigate to the {HubName} and click on *Execution Environments* if that window is not open.
```

Comment:
I would reword this to:
Log in to your automation hub.
From the navigation panel, select menu:Execution Environments[Execution Environments].